uwblueprint/sistering
# Skill +x tag and plural posting card bug fixes

## Ticket link

Closes #566 #571



## Implementation description
* admin posting card displays "1 volunteer" when singular instead of "volunteers"
* +x tag show all tags on hover instead of on click

<img width="540" alt="Screen Shot 2022-08-13 at 5 02 06 PM" src="https://user-images.githubusercontent.com/82241706/184734850-c4db9ea7-2427-416f-8ea2-5916e5d482cd.png">




## Steps to test
1. Sign in as a volunteer user and navigate to http://localhost:3000/edit-account, select more than 3 skills and languages and verify the "+x" tag works on hover
<img width="1344" alt="Screen Shot 2022-08-15 at 3 49 35 PM" src="https://user-images.githubusercontent.com/82241706/184733971-f8aa373a-68e3-4d8c-89fb-f93a20d512c9.png">

2. Sign in as an admin user, navigate to http://localhost:3000/admin/schedule/posting/1, select a time on the calendar with available volunteers and verify the skills +x tag works on hover
<img width="485" alt="Screen Shot 2022-08-15 at 3 53 06 PM" src="https://user-images.githubusercontent.com/82241706/184734217-346b1b03-03ed-41ea-811c-b6547419d77a.png">

3. Navigate to http://localhost:3000/admin/users and open the side panel for a volunteer user, verify the skills +x tag works on hover
<img width="661" alt="Screen Shot 2022-08-15 at 3 55 03 PM" src="https://user-images.githubusercontent.com/82241706/184734431-eae4fa9a-75bc-484c-aeef-d65d6095fcde.png">


## What should reviewers focus on?
* the behavior of the +x tags work as intended, the number displayed in the +x tag is correct


## Checklist
